bait-files/src/main/java/com/google/sps/servlets/UserHistoryServlet.java

```diff
@@ -47,26 +53,26 @@ public int compare(Review a, Review b) {
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     boolean isPendingReview = Boolean.parseBoolean(request.getParameter("is-pending-review"));
     if (isPendingReview) {
-      List<Review> pendingReviews = getReviewsByType(Status.PENDING);
+      List<Request> allRequests = ObjectifyService.ofy().load().type(Request.class).list();
+      List<Request> requests = allRequests.stream()
+                                   .filter(req -> req.userId.equals(userId))
+                                   .filter(req -> req.status == Status.PENDING)
+                                   .collect(toList());
+      Collections.sort(requests, new SortByEarliestDate());
```

Comment:
You can replace this with this sort
`    List<Review> reviews = allRequests.stream()
                               .filter(req -> req.userId.equals(userId))
                               .filter(req -> req.status == statusType)
                               .sorted(Comparator.comparing(req -> req.submissionDate)
                               .collect(toList());`

Reply:
If in the wrong direction you can add .reversed() to the comparator